deployments/rialto/docker-compose.yml

```diff
@@ -65,6 +65,20 @@ services:
       - bridge-node-bob
       - bridge-node-charlie
 
+  relay-eth-submit-exchange-tx:
```

Comment:
I'd rather call this `poa-exchange-tx-generator`, I know it uses `relay` binary to actually submit them, but I feel the `generator` part emphasizes that this thing is producing (test) transactions.